groundstation/backend_api/user.py

```diff
@@ -28,12 +28,21 @@ def get(self, user_id, local_args=None):
         :returns: response_object, status_code
         :rtype: tuple (dict, int)
         """
-        user = User.query.filter_by(id=user_id).first()
+        user = User.query.filter_by(id=User.decode_auth_token(user_id)).first()
```

Comment:
```suggestion
        user = User.query.filter_by(id=User.decode_auth_token(auth_token)).first()
```
In addition to this change, can you also change [Line 22](https://github.com/AlbertaSat/ex2_ground_station_website/blob/79509099bb70c42d5f36469092ebc3a193acd9db/groundstation/backend_api/user.py#L22) so that the parameter is `auth_token` instead of `user_id`?
```diff
- def get(self, user_id, local_args=None):
+ def get(self, auth_token, local_args=None):
```
and on [Line 182](https://github.com/AlbertaSat/ex2_ground_station_website/blob/79509099bb70c42d5f36469092ebc3a193acd9db/groundstation/backend_api/user.py#L182) change the url to match parameters:
```diff
- api.add_resource(UserEntity, '/api/users/<user_id>')
+ api.add_resource(UserEntity, '/api/users/<auth_token>')
```

These changes make it clearer what needs to be sent to the GET request